interface/wx/grid.h

```diff
@@ -22,7 +22,7 @@
          wxGridCellFloatRenderer, wxGridCellNumberRenderer,
          wxGridCellStringRenderer
 */
-class wxGridCellRenderer : public wxClientDataContainer, public wxRefCounter
+class wxGridCellRenderer : public wxSharedClientDataContainer, public wxRefCounter
```

Comment:
We need to document `wxSharedClientDataContainer` too if it appears in the documentation, it should be self-consistent.